chainer/functions/noise/gaussian.py

```diff
@@ -91,10 +94,35 @@ def gaussian(mean, ln_var):
         ln_var (~chainer.Variable):
             Input variable representing the logarithm of a variance
             :math:`\\log(\\sigma^2)`.
+        eps (ndarray or None):
+            The eps value to be used.
+            The shape and dtype must be the same as ``ln_var`` and should be
+            on the same device.
+            If ``eps`` is not specified or set to ``None``, an eps value will
```

Comment:
A suggestion but how about emphasizing that `eps` is "usually" not specified, but that it _can be_ if you want it to be deterministic.

Reply:
Fixed.